ui/redux/utils/reducerFactories.js

```diff
@@ -215,9 +215,15 @@ export const createObjectsByIdReducer = (updateActionType, initialState = {}, de
           console.error(`Invalid ${updateActionType} action. action.updatesById is undefined: `, action)
           return state
         }
-
+        let updates = action.updatesById
```

Comment:
what's the purpose of the key arg?

Reply:
It allows use to dispatch one `RECEIVE_DATA` action and have it update multiple different reducers by key, like `projectsByGuid`, `familiesByGuid`, etc